common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -193,6 +193,10 @@ def modulestore():
             settings.MODULESTORE['default'].get('OPTIONS', {})
         )
 
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX'):
+            from ccx.modulestore import CCXModulestoreWrapper
```

Comment:
Does this import need to be delayed to here? If `CCXModulestoreWrapper` is imported even if the feature is off, are there any adverse consequences?